package.json

```diff
@@ -29,21 +30,20 @@
     "e2e"
   ],
   "files": [
-    "dist",
+    "src/pure.js",
+    "src/index.js",
     "dont-cleanup-after-each.js",
     "pure.js",
```

Comment:
I don't think you need this since it's in `src` now. not sure what `dont-cleanup-after-each` is and whether you need that one or not
```suggestion
```

Reply:
Removed both. `dont-cleanup-after-each` seems to be a helper package for setting the env variable that is not used anywhere anyway.